generator/examples/simple/application_description.json

```diff
@@ -11,41 +11,123 @@
       "name": "service1",
       "clusters": [
         {
-          "cluster": "cluster-1",
-          "namespace": "default",
-          "node": "node-1"
+          "cluster": "cluster1",
+          "namespace": "edge-namespace",
```

Comment:
It is suggested to have the existing namespace for all cases, which is `default`.